tests/Fixtures/Entity/RichComment.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Fixtures\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class RichComment
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="text")
+     */
+    private $body;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=RichPost::class, inversedBy="comments")
+     * @ORM\JoinColumn(nullable=false, onDelete="CASCADE")
+     */
+    private $post;
+
+    public function __construct(string $body, RichPost $post)
+    {
+        $this->body = $body;
+        $this->post = $post;
+    }
+
+    public function getId(): ?int
```

Comment:
getId() return type should be `int`

Reply:
What about when the id truly is null (before the entity is persisted)?

Reply:
good point.

Reply:
if this entity is part of a "rich model" it should use uuids and set the uuid in its constructor :innocent: 

but I think we don't really care for these fixtures :shrug:

Reply:
This is very true! (I didn't want to add a dep)